draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -382,7 +405,110 @@
+          footprint types (respectively), for listing IP addresses blocks. Using Footprint Objects of these
+          types, one can define a FCI Capability Advertisement Object footprint constraints that match IPv4 
+          or IPv6 clients. However, the described "narrowing" semantic of the Footprint Objects array prevents
+          the usage of these objects together in order to create a footprint constraint that matches IPv4 
+          clients together with IPv6 clients. 
+       </t>
+       <t>Below is an example for an attempt creating an object matching 
+          IPv4 clients of subnet "192.0.2.0/24", as well as IPv6 clients of subnet "2001:db8::/32". 
+          Such definition results with an empty list of clients, as the constraints are additives and 
+          a client address cannot be both IPv4 and IPv6.
+      </t>
+
+        <figure>
+            <artwork><![CDATA[
+{
+  "capabilities": [
+    {
+      "capability-type": <CDNI capability object type>,
+      "capability-value": <CDNI capability object>,
+      },
+      "footprints": [
+          {
+              "footprint-type": "ipv4cidr",
+              "footprint-value": ["192.0.2.0/24"]
+          },
+          {
+              "footprint-type": "ipv6cidr",
+              "footprint-value": ["2001:db8::/32"]
+          }
+      ]
+    }
+  ]
+}
+]]>
+            </artwork>
+        </figure>
+
+
+      <t>To overcome the described limitation and allow a list of footprint constraints that
+          matches both IPv4 and IPv6 client addresses, below are specified the "IPv4v6CIDR" simple data type as well as
+          a derived footprint type.
+      </t>
+
+      <section anchor="cdni-metadata-ipv4v6cidr-data-type" title="CDNI Metadata IPv4v6CIDR Data Type">
+        <t>The "IPv4v6CIDR" data type specified in <xref target="cdni-metadata-ipv4v6cidr-data-type-description"/>, 
+            can be either IPv4 or IPv6 address blocks. The data type is appended to the list of data types
+            described in section 4.3 of <xref target="RFC8006"/> that are used as properties of CDNI Metadata objects.
+        </t>
+
+        <section anchor="cdni-metadata-ipv4v6cidr-data-type-description" title="CDNI Metadata IPv4v6CIDR Data Type Description">
+          <t>Either a IPv4CIDR property or IPv6CIDR property, as described in sections 4.3.5 and 4.3.6 of
+             <xref target="RFC8006"/> respectively.
+          </t>
+          <t><list style="empty">
+            <t>Type: String</t>
+            <t>Example IPv4v6CIDRs:</t>
+            <t>"192.0.2.0/24"</t>
+            <t>"2001:db8::/32"</t>
+          </list></t>
+        </section>
+      </section>
+
+      <section anchor="cdni-metadata-ipv4v6cidr-footprint-type-description" title="CDNI Metadata IPv4v6CIDR Footprint Type Description">
+        <t>The "IPv4v6CIDR" simple data type specified in <xref target="cdni-metadata-ipv4v6cidr-footprint-type"/>, is added
```

Comment:
I think you should change the reference here from the cdni-metadata-ipv4v6cidr-footprint-type tag to the cdni-metadata-ipv4v6cidr-data-type-description tag.

Reply:
ack